lib/omnibus/health_check.rb

```diff
@@ -151,6 +151,16 @@ class HealthCheck
       /libmd\.so/,
     ].freeze
 
+    IGNORED_ENDINGS = %w(
+    Gemfile LICENSE VERSION README Rakefile .[ch] .rdoc .js .json .gemspec .e*rb
+    .ri .lock .md .yml .txt .png .pm .py[oc]* .sh .toml .gitignore .r*html .log
+    .out .pl .lua .h*h .sql .ttf .xml .mkd .java
+    ).freeze
+
+    IGNORED_PATTERNS = %w(
+    /terminfo/ /share/doc/ /share/postgresql/ /share/terminfo/
+    ).freeze
```

Comment:
Let's place one array item per line (and in alpha order) so adding additional items to `IGNORED_ENDINGS` or `IGNORED_PATTERNS` in the future produces an easier to review git change. For example:

```ruby
IGNORED_PATTERNS = %w(
  /share/doc/
  /share/postgresql/
  /share/terminfo/
  /terminfo/ 
).freeze
```